packages/api/cms-api/src/mailer/mailer.service.ts

```diff
@@ -74,6 +75,9 @@ export class MailerService {
             await this.entityManager.flush();
         }
 
+        // Delete outdated logs, purposely not using await because it is not important for the mail sending process
+        this.mailerLogRepository.nativeDelete({ createdAt: { $lt: subDays(new Date(), this.mailerConfig.daysToKeepMailLog) } });
```

Comment:
does this work correctly? don't you need to commit the transaction (as flush does)? (If you want this in background, you might have to start a new transaction for that)

Reply:
I understand that nativeDelete directly executes a DELETE sql command. But additional to the unit-of-work we have a postgres transaction that needs a COMMIT command to apply changes.

does this work correctly, did you test it?